buzsaki_lab_to_nwb/huszar_hippocampus_dynamics/convert_session.py

```diff
@@ -25,14 +21,70 @@ def session_to_nwb(session_dir_path, output_dir_path, stub_test=False, verbose=F
     nwbfile_path = output_dir_path / f"{session_id}.nwb"
 
     source_data = dict()
+    conversion_options = dict()
+
     # Add sorter
     file_path = session_dir_path / f"{session_id}.spikes.cellinfo.mat"
     source_data.update(Sorting=dict(file_path=str(file_path), sampling_frequency=30_000.0))
 
     # Add behavior data
     source_data.update(Behavior8Maze=dict(folder_path=str(session_dir_path)))
+    conversion_options.update(Behavior8Maze=dict(stub_test=stub_test))
+
     source_data.update(BehaviorSleep=dict(folder_path=str(session_dir_path)))
 
+    # Add Recordings
+    file_path = session_dir_path / f"{session_id}.dat"
+    xml_file_path = session_dir_path / f"{session_id}.xml"
+
+    if file_path.is_file():
+        size_in_GB = file_path.stat().st_size / 1000
+        if size_in_GB:
+            if verbose:
+                print(f"The size of {file_path.name} is {size_in_GB} GB")
+            source_data.update(Recording=dict(file_path=str(file_path), xml_file_path=str(xml_file_path)))
+            conversion_options.update(
+                Recording=dict(stub_test=stub_test, write_electrical_series=write_electrical_series)
+            )
+        else:
+            print(f"Skipping recording interface for {session_id} because the file {file_path} does not have any data.")
+
+    else:
+        print(f"Skipping recording interface for {session_id} because the file {file_path} does not exist.")
+
+    # Add LFP
+    file_path = session_dir_path / f"{session_id}.lfp"
+    folder_path = session_dir_path
+    if file_path.is_file():
+        size_in_GB = file_path.stat().st_size / 1000**3
```

Comment:
Shoulnd't this be 1024**3? instead of 1000?

Reply:
Oh, I see, I think this that representation is OS dependent. It seems that linux and mac uses the decimal (1000) instead of the binary (1024) representation even it the latter is used for memory (RAM). 1000 makes more sense here.

Reply:
`st_size` just returns raw # of bytes

GB is the abbreviation for $10^9$

GiB is the abbreviation for $2^{30} = 1024^3$

so the name of the variable is accurate